go/vt/sqlparser/parse_test.go

```diff
@@ -1063,11 +1060,20 @@ var (
 			input:  "alter table a drop index idx",
 			output: "alter table a drop index idx",
 		}, {
-			input:  "alter table a add check ch_1",
-			output: "alter table a",
+			input:  "alter table a add constraint ch_1 check (b > 0)",
+			output: "alter table a add check (b > 0)",
```

Comment:
This isn't right. It's fine to normalize to a different statement form but you're dropping information here -- where did ch_1 go?

The print methods themselves aren't important, they're only used for these tests. But in the absence of a print method that gives a full textual representation of the AST, you need much more involved tests for the parser, so it's easier in most cases to just implement the printing.